api-specificatie/zrc/1.0.x/openapi.yaml

```diff
@@ -9521,6 +9537,12 @@ components:
           description: De datum waarop met de uitvoering van de zaak is gestart
           type: string
           minLength: 1
+        maximaleVertrouwelijkheidaanduiding:
+          title: Maximalevertrouwelijkheidaanduiding
+          description: Zaken met een vertrouwelijkheidaanduiding die beperkter is
+            dan de aangegeven aanduiding worden uit de resultaten gefiltered.
+          type: string
+          minLength: 1
```

Comment:
Ik zou hier `minLenght: 1` vervangen door de enum die ook eerder in de query parameter wordt gebruikt?
```
enum:
            - openbaar
            - beperkt_openbaar
            - intern
            - zaakvertrouwelijk
            - vertrouwelijk
            - confidentieel
            - geheim
            - zeer_geheim
```
Dan heb je het maximaal scherp gedefinieerd.

Reply:
ja, gaat blijkbaar een stukje parameter generatie niet helemaal lekker voor deze! ik ga het z.s.m. aanpassen

Reply:
De parameter generatie gaat juist wel goed, maar het euvel zit in de generatie van het schema van de request body van `POST zaak/_zoek`

Reply:
dat bedoelde ik - stukje die de `choices`/`enum` niet detecteert. gaat bij filters wel goed (moest ik ook patch), maar in de body niet dus :D

Reply:
bij deze gefixed - er zijn ook wat andere parameters bijgewerkt door deze bugfix.

Reply:
Mooie bijvangst!